content/en/user-guide/tools/localstack-docker-extension/index.md

```diff
@@ -8,50 +8,58 @@ aliases:
 
-<img src="localstack-docker-extension.png" alt="LocalStack Extension for Docker Desktop" title="LocalStack Extension for Docker Desktop" width="900" />
+<img src="localstack-docker-extension.png" alt="LocalStack Extension for Docker Desktop" title="LocalStack Extension for Docker Desktop" width="800px"   />
 
 ## Installation
 
-To utilize LocalStack's Docker Extension, it is necessary to have a recent version of Docker Desktop (v4.8 or higher) installed on the local machine. To enable the extension, access the **Preferences** tab and select the **Enable Docker Extensions** option within the **Extensions** tab.
+To utilize LocalStack's Docker Extension, it is necessary to have a recent version of Docker Desktop (v4.8 or higher) installed on the local machine. To enable the extension, access the **Extensions** tab and select the **Enable Docker Extensions** and **Show Docker Extensions system containers** option within the **Extensions** tab.
 
-<img src="localstack-docker-extension-preferences.png" title="Enable Docker Extensions in the Preferences within the Extensions tab" width="900" />
+<img src="localstack-docker-extension-preferences.png" title="Enable Docker Extensions in the Preferences within the Extensions tab" width="800px"   />
 <br></br>
 
 The LocalStack Extension for Docker Desktop has been validated and can be accessed on the Extensions Marketplace. To begin using it, navigate to the **Extensions Marketplace**, search for **LocalStack**, and click the **Install** button to proceed with the installation.
 
-<img src="localstack-docker-extension-marketplace.png" title="Discover the LocalStack Extension on the Docker Desktop Marketplace and install it!" width="900" />
+<img src="localstack-docker-extension-marketplace.png" title="Discover the LocalStack Extension on the Docker Desktop Marketplace and install it!" width="800px"  />
 <br></br>
 
 An alternative method for installing the LocalStack's Extension for Docker Desktop is pulling the [public Docker image](https://hub.docker.com/r/localstack/localstack-docker-desktop) from Docker Hub and installing it!
 
 {{< command >}}
-$ docker extension install localstack/localstack-docker-desktop:0.3.1
+$ docker extension install localstack/localstack-docker-desktop:0.5.3
 {{< /command >}}
 
 After installation, you can access the LocalStack Extension for Docker Desktop from the **Extensions** tab. Upon the initial launch of the extension, a prompt to select a mount point for the LocalStack container will appear. Select your username from the drop-down menu. Furthermore, you can modify this setting later by navigating to the **Configurations** tab and choosing a different mount point.
```

Comment:
The screenshot below shows the default "Shared" volume mount, maybe let's show the drop down expanded to show the options?